src/lib/markbind/package.json

```diff
@@ -27,6 +27,7 @@
     "markdown-it-emoji": "^1.4.0",
     "markdown-it-imsize": "^2.0.1",
     "markdown-it-ins": "^2.0.0",
+    "markdown-it-linkify-images": "^1.1.1",
```

Comment:
Shall we adjust the order of the sections in both package.json in this PR?
Or will it significantly mess up the diff since we're modifying packages too?

Ref: https://github.com/MarkBind/markbind/pull/1248#discussion_r439851727

Reply:
Hmm perhaps the one that would move `src/lib/markbind` into `packages/markbind`? Would be more ~suitable~ relevant I think
The diffs wouldn't be too big if we do it here either though

Reply:
I'm ok with either, if it doesn't significantly mess up the diff in this PR.

Reply:
👍 I'll give it a try, will stash it if it's too much

Reply:
The diffs seem ok (Fixup hoist markbind core package dependencies commit); I extracted out the `jest` config into a separate file for consistency as well

Reply:
https://github.com/MarkBind/markbind/pull/1253/commits/4da36b4f85ab37ac0439cf0122cdb394673ddd3a should be its own commit instead of a fixup.
Otherwise it messes up the diff since it's no longer obvious that we removed several packages.